### PR TITLE
Add badges

### PR DESCRIPTION
**Pull request by qinwf-nuan**
## Description

Add badges and update readme.

Travis CI macOS server is down for now, so the the macOS build failed.

see https://www.traviscistatus.com/

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -48,6 +46,7 @@ npm install
 + [x] Matrix API
 + [x] Model API
 + [x] Prediction API
++ [ ] Windows Support
```
Comment (jiabinf) [inline readme.md]: move `Windows Support` below `Async API`?

**Comment by jiabinf** [comment]
ping

File: readme.md (hunk 1 above)
Comment (qinwf-nuan) [reply readme.md]: updated.

**Comment by jiabinf** [comment]
LGTM

**Pull request merged**